### PR TITLE
Added Imagenet classes file

### PR DESCRIPTION
**Pull request by georgepachitariu**

### CODE REVIEW

File: imagenet_classes.csv [hunk 1]
```diff
@@ -0,0 +1,1000 @@
+0tench, Tinca tinca
```
Comment (georgepachitariu) [inline imagenet_classes.csv:1]: In between the index (0) and label (tench, Tinca tinca) there is the non-printable character "\001", that I use to split the field with in the Python code.

**Comment by stes** [comment]
Hi @georgepachitariu is this still needed, or can we close?

**Comment by georgepachitariu** [comment]
It's still needed @stes. The original file cannot be loaded in python without having to parse every line.

**Comment by stes** [comment]
all right, merged then @georgepachitariu

**Pull request merged**